ui/src/flux/components/TimeMachineEditor.tsx

```diff
@@ -98,23 +108,55 @@ class TimeMachineEditor extends PureComponent<Props> {
     this.editor.clearGutter('error-gutter')
     const lineNumbers = this.statusLine
     lineNumbers.forEach(({line, text}) => {
+      const lineNumber = line - 1
       this.editor.setGutterMarker(
-        line - 1,
+        lineNumber,
         'error-gutter',
-        this.errorMarker(text)
+        this.errorMarker(text, lineNumber)
       )
     })
 
     this.editor.refresh()
   }
 
-  private errorMarker(message: string): HTMLElement {
+  private errorMarker(message: string, line: number): HTMLElement {
     const span = document.createElement('span')
     span.className = 'icon stop error-warning'
     span.title = message
+    span.addEventListener('click', this.handleClickError(message, line))
```

Comment:
Does this ever get removed? Worried about memory leaks.

Reply:
The error `<span>`s (and the event listeners belonging to them) get cleared when there are no longer any errors.